proxy/proxy.dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM nginxinc/nginx-unprivileged:1-alpine
 
-COPY ./default.conf /etc/nginx/conf.d/default.conf
-COPY ./uwsgi_params /etc/nginx/uwsgi_params
+COPY ./proxy/default.conf.template /etc/nginx/templates/default.conf.template
```

Comment:
Shouldn't this be renamed to `default.conf`?

Reply:
No, the `default.conf` file will be auto generated using `envsubst` replacing the values of the template  `default.conf.template` with the actual values in the environment variables.

> More information here https://hub.docker.com/_/nginx